src/components/organize/journeys/JourneyInstanceSummary.tsx

```diff
@@ -93,6 +93,12 @@ const JourneyInstanceSummary = ({
           ref={editingRef}
           data-testid="JourneyInstanceSummary-textArea"
           onChange={(value) => setSummary(value)}
+          placeholder={intl.formatMessage(
+            {
+              id: 'pages.organizeJourneyInstance.summaryPlaceholder',
+            },
+            { journeyTitle: journeyInstance.journey.title.toLowerCase() }
```

Comment:
This is one of those cases where we shouldn't be using the title, but the `singular_label` that unfortunately is not available on these objects in the API. This will need to be replaced once it's been added in the API, as tracked by #677.

Reply:
I was thinking about this too!

Reply:
Thank you for the clarification above!! :D